data/json/npcs/exodii/exodii_merchant_talk.json

```
@@ -638,6 +645,27 @@
       { "text": "Well, I'd better be going.  Bye.", "topic": "TALK_DONE" }
     ]
   },
+  {
+    "id": "TALK_EXODII_MERCHANT_Jobs",
+    "type": "talk_topic",
+    "dynamic_line": "&Rubik pulls out a binder, and flips through the pages for a second.  \"Yeah, we're always in need of some help around here.  Here's the list I've been keeping for stuff like this, what are you interested in?\"",
```

```suggestion
    "dynamic_line": "*Rubik pulls out a sheaf of mismatched papers, and leafs through the pages for a second.  Their broad metallic fingers look surprisingly deft and delicate, and incongruous on the otherwise unremarkable looseleaf.  \"Aye, an' there's room for a pop-an-work roun' here.  Rubik's got tassed a lissy, right and clean.",
    "//~": "Yeah, we're always in need of some help around here.  Here's the list I've got for stuff like this."
```
We're going to want to also add a conditional dynamic line here so Rubik can loosely describe available mission(s) for you.